components/data-table/index.jsx

```diff
@@ -451,82 +483,103 @@ class DataTable extends React.Component {
+						assistiveText={assistiveText}
+						allSelected={allSelected}
+						fixedHeader={this.props.fixedHeader}
+						headerRefs={(ref, index) => {
+							if (index === 'action' || index === 'select') {
+								if (ref) {
+									this.headerRefs[index][0] = ref;
+								} else {
+									this.headerRefs[index] = [];
+								}
 							} else {
-								this.headerRefs[index] = [];
+								this.headerRefs.column[index] = ref;
 							}
-						} else {
-							this.headerRefs.column[index] = ref;
-						}
-					}}
-					indeterminateSelected={indeterminateSelected}
-					canSelectRows={canSelectRows}
-					columns={columns}
-					id={`${this.getId()}-${DATA_TABLE_HEAD}`}
-					onToggleAll={this.handleToggleAll}
-					onSort={this.props.onSort}
-					showRowActions={!!RowActions}
-				/>
-				<tbody>
-					{numRows > 0
-						? this.props.items.map((item, index) => {
-								const rowId =
-									this.getId() && item.id
-										? `${this.getId()}-${DATA_TABLE_ROW}-${item.id}`
-										: shortid.generate();
-								return (
-									<DataTableRow
-										assistiveText={assistiveText}
-										canSelectRows={canSelectRows}
-										className={item.classNameRow}
-										columns={columns}
-										fixedLayout={this.props.fixedLayout}
-										id={rowId}
-										index={index}
-										item={item}
-										key={rowId}
-										onToggle={this.handleRowToggle}
-										selection={this.props.selection}
-										rowActions={RowActions}
-										tableId={this.getId()}
-									/>
-								);
-						  })
-						: // Someday this should be an element to render when the table is empty
-						  null}
-				</tbody>
-			</table>
+						}}
+						indeterminateSelected={indeterminateSelected}
+						canSelectRows={canSelectRows}
+						columns={columns}
+						id={`${this.getId()}-${DATA_TABLE_HEAD}`}
+						onToggleAll={this.handleToggleAll}
+						onSort={this.props.onSort}
+						showRowActions={!!RowActions}
+					/>
+					<tbody>
+						{numRows > 0
+							? this.props.items.map((item, index) => {
+									const rowId =
+										this.getId() && item.id
+											? `${this.getId()}-${DATA_TABLE_ROW}-${item.id}`
+											: shortid.generate();
+									return (
+										<DataTableRow
+											assistiveText={assistiveText}
+											canSelectRows={canSelectRows}
+											className={item.classNameRow}
+											columns={columns}
+											fixedLayout={this.props.fixedLayout}
+											id={rowId}
+											index={index}
+											item={item}
+											key={rowId}
+											onToggle={this.handleRowToggle}
+											selection={this.props.selection}
+											rowActions={RowActions}
+											tableId={this.getId()}
+										/>
+									);
+							  })
+							: // Someday this should be an element to render when the table is empty
+							  null}
+					</tbody>
+				</table>
+				{this.props.fixedHeader && this.props.hasMore && (
+					<div
+						style={{
+							height: '3rem',
```

Comment:
Instead of inline styles, it would be better if we could find SLDS classes that we could use, like in [Grid](http://lightningdesignsystem.com/utilities/grid/).

Reply:
There wasn't any SLDS markup for this so I couldn't see how they styled this.  I was trying to look at other components to see how they styled things but didn't see much usage of slds class names for things like this.

Reply:
How about something like:
```
<div class="slds-grid slds-gutters">
  <div class="slds-col">
    <span>spinner</span>
  </div>
</div>```